src/django_htmx/http.py

```diff
@@ -26,6 +26,32 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self._reason_phrase = "Stop Polling"
 
 
+class HttpResponseLocation(HttpResponseRedirectBase):
+    status_code = 200
+    swap_spec_kwargs = (
+        "source",
+        "event",
+        "handler",
+        "target",
+        "swap",
+        "values",
+        "headers",
+    )
+
+    def __init__(self, redirect_to: str, *args: Any, **kwargs: Any) -> None:
+        swap_details = {}
+        for key in self.swap_spec_kwargs:
+            if key in kwargs:
+                swap_details[key] = kwargs.pop(key)
```

Comment:
(rather than take arbitrary kwargs and then extract a limited set, declare arguments explicitly, so that type hints can help users define them correctly)

Reply:
I did that and found myself using Any for a couple of them - I need to double check headers/values to remember what that should be